### PR TITLE
updates to some module screenshots

### PR DESCRIPTION
**Pull request by tobes**
some updates to module screenshots

plus missing color options for xscreensaver

### CODE REVIEW

File: py3status/modules/netdata.py [hunk 1]
```diff
@@ -28,8 +28,8 @@
 
 SAMPLE OUTPUT
 [
-    {'full_text': 'eth0'},
-    {'color': '#FF0000', 'full_text': 'LAN(Kb):  16.9↓   5.2↑', 'separator': True},
+    {'full_text': 'eth0 '},
```
Comment (ultrabug) [inline py3status/modules/netdata.py]: why the trailing space plz ?

Comment (lasers) [inline py3status/modules/netdata.py]: We merged everything into one format for `netdata.py` not long ago. Here, we replace separator with a space. The `keyboard_locks.py` typo change in this PR also contain trailing spaces.

Comment (tobes) [inline py3status/modules/netdata.py]: `eth0LAN(Kb)...` vs `eth0 LAN(Kb)...`

Comment (lasers) [inline py3status/modules/netdata.py]: We don't paint both of them `red`
```
[
{'full_text': 'eno1 '},
{'full_text': 'LAN(Kb):  77.8↓  26.9↑ ', 'color': '#00FF00'},
{'full_text': 'T(Mb): 394↓  45↑ 438↕', 'color': '#FFFF00'},
]
```

Comment (tobes) [inline py3status/modules/netdata.py]: ok updated but I kept `eth0` as it is a more traditional name

File: py3status/modules/netdata.py [hunk 2]
```diff
@@ -28,9 +28,12 @@
...
-    {'full_text': 'eth0'},
-    {'color': '#FF0000', 'full_text': 'LAN(Kb):  16.9↓   5.2↑', 'separator': True},
-    {'color': '#FF0000', 'full_text': 'T(Mb): 2264↓ 143↑ 2407↕'}
+    {'full_text': 'eth0 '},
+    {'full_text': 'LAN(Kb):  77.8↓  26.9↑ ', 'color': '#00FF00'},
+    {'full_text': 'T(Mb): 394↓  45↑ 438↕', 'color': '#FFFF00'},
+]
+
+
 ]
```
Comment (lasers) [inline py3status/modules/netdata.py]: Extra `]`.

Comment (tobes) [inline py3status/modules/netdata.py]: Good spot 👍

**Comment by ultrabug** [comment]
Thanks !

**Pull request merged**